curriculum/challenges/english/25-front-end-development/quiz-typescript/66f1ae758b77cfc3e4da6151.md

```diff
@@ -17,439 +17,454 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+What is Typescript?
 
 #### --distractors--
 
-Placeholder distractor 1
+A framework used for system programming.
 
 ---
 
-Placeholder distractor 2
+A database management tool.
 
 ---
 
-Placeholder distractor 3
+A programming language used for styling websites.
 
 #### --answer--
 
-Placeholder answer
+A superset of JavaScript that adds static typing.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Why should TypeScript be used in a project?
 
 #### --distractors--
 
-Placeholder distractor 1
+Eliminates need for testing in applications.
 
 ---
 
-Placeholder distractor 2
+It is faster than all other programming languages.
 
 ---
 
-Placeholder distractor 3
+It makes styling websites easier.
 
 #### --answer--
 
-Placeholder answer
+Improves maintainability by adding type definitions to JavaScript.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What language is TypeScript compiled down to?
 
 #### --distractors--
 
-Placeholder distractor 1
+Java
 
 ---
 
-Placeholder distractor 2
+Python
 
 ---
 
-Placeholder distractor 3
+C++
 
 #### --answer--
-
-Placeholder answer
+JavaScript
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+    function sum(a: number, b: number): number {
+	    return a + b;
+	}
+	console.log(sum(2, 3));
+
```

Comment:
code blocks need to be fenced, and here you also need words for your question, a code snippet is not enough to understand what it is being asked
````suggestion
Question using words goes here

```ts
    function sum(a: number, b: number): number {
	    return a + b;
	}
	console.log(sum(2, 3));

```
````

Reply:
The code needs to be formatted as well.

Reply:
Fixed the formatting. Thanks for pointing it out. @lasjorg